docs/source/cpp/compute.rst

```diff
@@ -181,7 +181,8 @@ of general type categories:
 
 * "String-like": String, LargeString.
 
-* "List-like": List, LargeList, sometimes also FixedSizeList.
+* "List-like": List, LargeList, ListView, LargeListView, and sometimes also
+  FixedSizeList.
```

Comment:
We should also fix/update the wording of the description for the `list_parent_indices` function. Specifically, the sentence _"Nulls in the parent list array are discarded"_ has become incorrect for list views.

Reply:
Perhaps _"Null is emitted for values in the child array that are not referenced by the parent list or list view array, for example because of a null in the list array"_?

Reply:
Looks like even the old description is wrong. Indices of a non-empty list entry would still be represented on the output of `list_parent_indices`.

I can't do the same for list-views because (1) it's more complicated than that, and (2) it wouldn't guarantee full coverage anyways.

I will update the note.

Reply:
Thanks!